calyx-py/calyx/builder.py

```diff
@@ -132,18 +132,6 @@ def output(
         """
         return self._port_with_attributes(name, size, False, attribute_literals)
 
-    def output_with_attributes(
-        self,
-        name: str,
-        size: int,
-        attribute_literals: List[Union[str, Tuple[str, int]]],
-    ) -> ExprBuilder:
-        """Declare an output port on the component with attributes.
-
-        Returns an expression builder for the port.
-        """
-        return self._port_with_attributes(name, size, False, attribute_literals)
-
```

Comment:
This is some unrelated cleanup, the `output` method already exists and does the same thing